### ISSUE TITLE
Ad-hoc star voting

### ISSUE THREAD

**Issue by elsirion** [MakePrisms/bitcoinmints#22]
When looking at the published recommendation events I noticed that the star voting system of bitcoinmints.com is implemented in a very ad-hoc way. Instead of adding another optional tag to contain the star voting or making the content JSON encoded you just serialize a `[x/5]` prefix in front of the comment. I think it would be good to standardize some sort of voting system into [NIP-87](https://github.com/nostr-protocol/nips/pull/1110) to ensure compatibility. cc @benthecarman

**Comment by gudnuf** [comment]
agreed, I tried to get some sort of "rating" tag into NIP-87, but everyone seemed to be against it. My reasoning for putting it in the content was so that clients that don't support the rating could still see it, but would be better to have as a tag.
